### ISSUE TITLE
Tray Icon? Hidden startup at login with no dock

### ISSUE THREAD

**Issue by mrunc** [overtake/TelegramSwift#101]
Tray Icon? Hidden startup at login with no dock or window
Any plans for these enhancements? it's a must for anyone using Telegram for notifications and bots

**Comment by eivindml** [comment]
Yes. This would be a great feature. :)

**Comment by gazzerh** [comment]
Surprised a tray icon hasn't been implemented yet. Tempted to use the old client for this reason.

**Comment by peterlupu** [comment]
yes, I definitely miss the tray icon as I have autohide dock and sometimes miss messages because I don't see the 'unread' message number in the tray. Any reason why this isn't an option in the Swift version?
